mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLHostOps.td

```diff
@@ -228,4 +228,30 @@ def SYCLHostScheduleKernel : SYCL_HostOp<"schedule_kernel",
   let hasVerifier = 1;
 }
 
+def SYCLHostSubmitOp : SYCL_HostOp<"submit",
+    [DeclareOpInterfaceMethods<SymbolUserOpInterface>]> {
+  let summary = "Submits a SYCL kernel.";
+  let description = [{
+    This operation represents the submission of a SYCL kernel
+    (`sycl::queue::submit`). In addition to the function implementing the CGF,
```

Comment:
CGF ? What do you mean by that acronym here ?

Reply:
What are the properties the CGF function must have ?  Does it have to be a llvm.func, with private linkage or not, what signature is expected ?

Reply:
I added more checks